freqtrade/freqtradebot.py

```diff
@@ -337,9 +337,7 @@ def _get_trade_stake_amount(self, pair) -> Optional[float]:
         else:
             stake_amount = self.config['stake_amount']
 
-        # TODO: should come from the wallet
-        avaliable_amount = self.exchange.get_balance(self.config['stake_currency'])
```

Comment:
i think `get_balance` could be removed from the exchange class completely, it's not called anymore (rpc uses `get_balances` - so that would simplify that class slightly, and remove 2 tests ... ;)

Reply:
probably in a different PR ?